src/Api/Serializer/RestSerializer.php

```diff
@@ -126,6 +126,15 @@ private function applyHeader($name, Shape $member, $value, array &$opts)
         if ($member->getType() == 'timestamp') {
             $value = TimestampShape::format($value, 'rfc822');
         }
+        if ($member['jsonvalue']) {
+            $value = json_encode($value);
+            if ($value === null && JSON_ERROR_NONE !== json_last_error()) {
+                throw new \ParserException('Unable to encode the provided value'
```

Comment:
Do you have any tests exercising this error path? There's no standard `ParserException` in PHP, and I don't think the SDK declares any exceptions in the root namespace.